python/openmldb_tool/diagnostic_tool/diagnose.py

```diff
@@ -114,35 +117,60 @@ def insepct_online(args):
     assert not fails, f"unhealthy tables: {fails}"
     print(f"all tables are healthy")
 
-    if hasattr(args, 'dist'):
-        if args.dist:
-            table_checker = TableChecker(conn)
-            table_checker.check_distribution(dbs=flags.FLAGS.db.split(","))
+    if getattr(args, 'dist', False):
+        table_checker = TableChecker(conn)
+        table_checker.check_distribution(dbs=flags.FLAGS.db.split(","))
 
 
 def inspect_offline(args):
     """scan jobs status, show job log if failed"""
+    final_failed = "failed,killed,lost"
```

Comment:
string is not a clear way. you can use array ['failed','killed','lost']
and in _get_jobs, it could be
`if no states or row[2].lower() in states]`